lib/solarwinds_apm/inst/graphql.rb

```diff
@@ -4,247 +4,202 @@
   module GraphQL
     module Tracing
-      # AppOpticsTracing in the graphql gem may be a different version than the
-      # one defined here, we want to use the newer one
-      redefine = true
-      this_version = Gem::Version.new('1.1.0')
-
-      if defined?(GraphQL::Tracing::AppOpticsTracing)
-        if this_version > GraphQL::Tracing::AppOpticsTracing.version
-          send(:remove_const, :AppOpticsTracing)
-        else
-          redefine = false
-        end
-      end
 
-      # TODO remove redefine for rebranded nighthawk
-      #  there will be no code in the graphql gem for it
-
-      if redefine
-        #-----------------------------------------------------------------------------#
-        #----- this class is duplicated in the graphql gem ---------------------------#
-        #-----------------------------------------------------------------------------#
-        class AppOpticsTracing < GraphQL::Tracing::PlatformTracing
-          # These GraphQL events will show up as 'graphql.prep' spans
-          PREP_KEYS = ['lex', 'parse', 'validate', 'analyze_query', 'analyze_multiplex'].freeze
-          EXEC_KEYS = ['execute_multiplex', 'execute_query', 'execute_query_lazy'].freeze
-
-          # During auto-instrumentation this version of AppOpticsTracing is compared
-          # with the version provided in the graphql gem, so that the newer
-          # version of the class can be used
-
-          # TODO remove for rebranded nighthawk
-          def self.version
-            Gem::Version.new('1.1.0')
-          end
+      class SolarWindsAPMTracing < GraphQL::Tracing::PlatformTracing
+        # These GraphQL events will show up as 'graphql.prep' spans
+        PREP_KEYS = ['lex', 'parse', 'validate', 'analyze_query', 'analyze_multiplex'].freeze
+        EXEC_KEYS = ['execute_multiplex', 'execute_query', 'execute_query_lazy'].freeze
 
-          self.platform_keys = {
-            'lex' => 'lex',
-            'parse' => 'parse',
-            'validate' => 'validate',
-            'analyze_query' => 'analyze_query',
-            'analyze_multiplex' => 'analyze_multiplex',
-            'execute_multiplex' => 'execute_multiplex',
-            'execute_query' => 'execute_query',
-            'execute_query_lazy' => 'execute_query_lazy'
-          }
-
-          def platform_trace(platform_key, _key, data)
-            return yield if !defined?(SolarWindsAPM) || gql_config[:enabled] == false
-
-            layer = span_name(platform_key)
-            kvs = metadata(data, layer)
-            kvs[:Key] = platform_key if (PREP_KEYS + EXEC_KEYS).include?(platform_key)
-
-            transaction_name(kvs[:InboundQuery]) if kvs[:InboundQuery] && layer == 'graphql.execute'
-
-            ::SolarWindsAPM::SDK.trace(layer, kvs: kvs) do
-              kvs.clear # we don't have to send them twice
-              yield
-            end
-          end
+        self.platform_keys = {
```

Comment:
Minor and a tangent: Is `self.platform_keys` used? Doing a grep for `platform_keys` within this repo doesn't give any results.

Reply:
It is part of the graphql api

Reply:
Ahhh ok thanks!